detection_rules/devtools.py

```diff
@@ -147,6 +152,43 @@ def build_integration_docs(ctx: click.Context, registry_version: str, pre: str,
     return docs
 
 
+@dev_group.command("bump-versions")
+@click.option("--major", is_flag=True, help="bump the major version")
+@click.option("--minor", is_flag=True, help="bump the minor version")
+@click.option("--patch", is_flag=True, help="bump the patch version")
+@click.option("--save", is_flag=True, help="Update the setup.cfg and packages.yml file")
+def bump_versions(major, minor, patch, save):
+    """Bump the versions"""
+
+    package = load_etc_dump('packages.yml')['package']
+    ver = package["name"]
+    new_version = Version(ver).bump(major, minor, patch)
+
+    kibana_version = f"^{new_version}.0" if not patch else "^{new_version}"
+    registry_version = f"{new_version}.0-dev.0" if not patch else f"{new_version}-dev.0"
+
+    # print the new versions
+    click.echo(f"New package version: {new_version}")
+    click.echo(f"New registry data version: {registry_version}")
+    click.echo(f"New Kibana version: {kibana_version}")
+    package["name"] = str(new_version)
+    package["registry_data"]["conditions"]["kibana.version"] = kibana_version
+    package["registry_data"]["version"] = registry_version
+
+    if save:
+        # update packages.yml
+        save_etc_dump({"package": package}, "packages.yml")
```

Comment:
This will remove comments.

Reply:
We may need to revisit this once we move to Package Storage v2 as the trailing strings will be important for automated promotion in GCP buckets.